feature-staking-api/src/main/java/io/novafoundation/nova/feature_staking_api/data/dashboard/common/ChainExt.kt

```diff
@@ -7,6 +7,14 @@ import io.novafoundation.nova.runtime.multiNetwork.ChainsById
 import io.novafoundation.nova.runtime.multiNetwork.chain.model.Chain
 import io.novafoundation.nova.runtime.multiNetwork.findChains
 import io.novafoundation.nova.runtime.multiNetwork.findChainsById
+import kotlinx.coroutines.flow.Flow
+import kotlinx.coroutines.flow.map
+
+fun ChainRegistry.stakingChainsFlow(): Flow<List<Chain>> {
+    return currentChains.map { chains ->
+        chains.filter { it.utilityAsset.supportedStakingOptions().isNotEmpty() }
```

Comment:
We can extract ` chain.utilityAsset.supportedStakingOptions().isNotEmpty()` condition into separate function since it is used twice